engine/gamesys/src/gamesys/components/comp_camera.cpp

```diff
@@ -124,6 +118,50 @@ namespace dmGameSystem
         return dmGameObject::CREATE_RESULT_OK;
     }
 
+    static inline dmMessage::URL CameraToURL(const CameraComponent* camera)
+    {
+        dmMessage::URL camera_url;
+        camera_url.m_Socket = dmGameObject::GetMessageSocket(dmGameObject::GetCollection(camera->m_Instance));
+        camera_url.m_Path   = dmGameObject::GetIdentifier(camera->m_Instance);
+        dmGameObject::GetComponentId(camera->m_Instance, camera->m_ComponentIndex, &camera_url.m_Fragment);
+        return camera_url;
+    }
+
+    static void CameraStackPop(CameraWorld* world, CameraComponent* camera)
```

Comment:
We still use a camera stack because we keep track of a "main" camera based on the order for the stack. If the current main camera gets removed we need to know which was the previous main camera so we can put that into the camera.MAIN property